src/color_laws.jl

```diff
@@ -1,6 +1,3 @@
-using BSplineKit: interpolate, BSplineOrder, Natural
-using DelimitedFiles
```

Comment:
I think https://github.com/JuliaAstro/DustExtinction.jl/pull/54/files#diff-9880a77dc8047f68a334d57b03fdc90fa3d1fcc9898d313378cc414f32f077eeL265 makes this stale now, so I removed it. Not sure yet why Aqua is complaining about this